plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -816,6 +817,17 @@ public void testNativeQueryInvalidArgument()
                 "Can't parse 'filter' argument as json");
     }
 
+    @Test
+    public void testNativeQueryWithCaseInSensitiveNameMatch()
```

Comment:
Please move before `testNativeQueryCaseNonLowercaseColumn`.

Reply:
> Please move before `testNativeQueryCaseNonLowercaseColumn`.

I moved it before `testNativeQueryCaseNonLowercaseColumn`.